apps/chat/src/components/Common/Buttons/ToggleSidebarButtor.tsx

```diff
@@ -33,26 +35,35 @@ export const ToggleSidebarButton: React.FC<Props> = ({
     onToggle();
   };
 
+  const ToggleIcon = useCallback(() => {
+    const iconProps = {
+      className: 'text-secondary hover:text-accent-primary',
+      width: iconSize,
+      height: iconSize,
+    };
+
+    if (isOpened)
+      return rightSide ? (
+        <MoveRightIcon {...iconProps} />
+      ) : (
+        <MoveLeftIcon {...iconProps} />
+      );
+
+    return rightSide ? (
+      <MoveLeftIcon {...iconProps} />
+    ) : (
+      <MoveRightIcon {...iconProps} />
+    );
+  }, [iconSize, isOpened, rightSide]);
```

Comment:
need to use `rotate-180` for right side
```suggestion
  const Icon = isOpened ? MoveLeftIcon : MoveRightIcon;
```